index.html

```diff
@@ -25,14 +25,14 @@
   </header>
 
   <section class="headline">
-    <h1>I'm Alexandra <br> Glad to see you!</h1>
+    <h1 class="name">I'm Alexandra <br> Glad to see you!</h1>
     <p>
       I’m AlexandraGlad to see you!I’m a software developer! 
       I can help you build a product , feature or website Look 
       through some of my work and experience! If you like what you 
       see and have a project you need coded, don’t hestiate to contact me.
     </p>
-      <a href="" class="connect">
+      <a href="#" class="connect">
       Let's Connect
     </a>
```

Comment:
- [Optional] Please make sure that your design looks exactly like Figma design, I know it is stressful to just make the 100% design match but it is a really valuable skill that all companies search for please fix the sizing in the headline section I suggest you use (VW, it calculate the size depending on the page width) in all your design (width, height, font-size, margin,....) you can use this [website](https://web-development.space/tools/px-to-vw/) to compare all the sizes in the design with your website 
![image](https://user-images.githubusercontent.com/74744787/208268208-8fac4c29-d544-4134-b82d-ae9cdb22de49.png)